tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -118,9 +247,9 @@ std::string getFusionFullName(lmhlo::FusionOp op) {
   return (name_attr.getValue() + Twine("___") + tag_attr.getValue()).str();
 }
 
-std::string generateSignatureForFusion(lmhlo::FusionOp op) {
+std::string generateSignatureForFusion(lmhlo::FusionOp op,
```

Comment:
Do we actually use the `op` parameter? Maybe use `fusionPattern ` is enough. We can also get the op list from `fusionPattern`.

Reply:
To check.